docs/SUMMARY.md

```diff
@@ -10,7 +10,7 @@
     * [MLIP-000](doc/proposals/mlip-000.md)
   * [spark](doc/spark.md)
 * [LICENSE](license.md)
-* [MAINTAINERS](maintainers.md)
-* [CODE\_OF\_CONDUCT](code_of_conduct.md)
-* [CONTRIBUTING](contributing.md)
+* [MAINTAINERS](MAINTAINERS.md)
+* [CODE\_OF\_CONDUCT](CODE_OF_CONDUCT.md)
+* [CONTRIBUTING](CONTRIBUTING.md)
```

Comment:
```suggestion
* [CONTRIBUTING](docs/CONTRIBUTING.md)
```

Reply:
Should I also remove from the summary 
```
* [doc](doc/README.md)
  * [neural\_splitter\_arch](doc/neural_splitter_arch.md)
  * [topic\_modeling](doc/topic_modeling.md)
  * [cmd](doc/cmd/README.md)
    * [Preprocrepos command](doc/cmd/preprocrepos.md)
  * [README](doc/proposals/README.md)
    * [MLIP-000](doc/proposals/mlip-000.md)
  * [spark](doc/spark.md)
```

(Specially mentions to spark)

Reply:
I removed all the links to documentation that is not yet in the project to avoid dead links. 

If you are ok with it, I will add it back then I do the PR for docs.